### PR TITLE
small change to commitabletransaction sample

### PR DESCRIPTION
**Pull request by julielerman**
moved openconnection to happen before attempting to enlisttransactions

### CODE REVIEW

**Comment by divega** [comment]
See https://github.com/aspnet/EntityFramework.Docs/issues/738#issuecomment-394769747.

File: samples/core/Saving/Saving/Transactions/CommitableTransaction/Sample.cs [hunk 1]
```diff
@@ -33,9 +33,9 @@ public static void Run()
 
                     using (var context = new BloggingContext(options))
                     {
-                        context.Database.EnlistTransaction(transaction);
                         context.Database.OpenConnection();
-
+                        context.Database.EnlistTransaction(transaction);
```
Comment (AndriySvyryd) [inline samples/core/Saving/Saving/Transactions/CommitableTransaction/Sample.cs]: You should also add `context.Database.CloseConnection();` at the end

**Comment by divega** [comment]
Thanks @julielerman!

**Comment by julielerman** [comment]
itty bitty teeny tiny contribution ;)

File: samples/core/Saving/Saving/Transactions/CommitableTransaction/Sample.cs [hunk 2]
```diff
@@ -44,6 +44,7 @@ public static void Run()
                         // Run an EF Core command in the transaction
                         context.Blogs.Add(new Blog { Url = "http://blogs.msdn.com/dotnet" });
                         context.SaveChanges();
+                        Adding context.Database.CloseConnection();
```
[the remark this answers is not in the document]
Comment (divega) [reply samples/core/Saving/Saving/Transactions/CommitableTransaction/Sample.cs]: That was me :smile:

[the remark this answers is not in the document]
Comment (divega) [reply samples/core/Saving/Saving/Transactions/CommitableTransaction/Sample.cs]: Fixed committed to master.

**Pull request merged**